src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -94,7 +94,12 @@ public sealed class Features
 
         internal static readonly Features None = new Features();
 
-        public static readonly Features PowerFxV1 = new Features
+        /// <summary>
+        /// The default V1 Power Fx feature set. 
+        /// </summary>
+        public static Features PowerFxV1 => _powerFxV1;
```

Comment:
Keep this as a property,  #2740 changed it to a field, which is a binary breaking change.